### PR TITLE
fix_integers

### PR DESCRIPTION
**Pull request by dkuku**
I had a problem with I think this line:
```
order by 1, 2

```
It was due to this function below only accepting 3 element tuples but integer(min: 1) was returning integer.
```
  def __as_sql_language__({ttype, meta, value}) do
    {ttype, Map.put(meta, :language, :sql), value}
  end
```
I was using this file:
https://github.com/dbcli/pgspecial/blob/616f1df83faf7be7ee3e4f07a12cf0cf963ca9a9/dbcommands.sql

### CODE REVIEW

File: lib/makeup_sql.ex [hunk 1]
```diff
@@ -1010,9 +1007,19 @@ defmodule MakeupSql do
...
+  whitespace = ascii_string(@known_whitespace_characters, min: 1) |> token(:whitespace)
   newline = ascii_char([?\r, ?\n])
 
+  keyword_constants =
+    word_from_list(
+      ["TRUE", "FALSE", "NULL", "true", "false", "null"],
+      :keyword_constant
+    )
+
+  any_char = utf8_char([]) |> token(:error)
```
Comment (dkuku) [inline lib/makeup_sql.ex:1021]: It's from makeup source code

Comment (dkuku) [reply lib/makeup_sql.ex:1021]: Hey @Billzabob . Do you plan to maintain this package ?

Comment (Billzabob) [reply lib/makeup_sql.ex:1021]: Oh wow, totally forgot about this. My bad. Ya I don’t mind maintaining it. Seems like a pretty low maintenance library anyways. I’ll get this merged and published for you tomorrow 👍

Comment (Billzabob) [reply lib/makeup_sql.ex:1021]: Published! Thanks for the reminder and sorry for taking a year to get this out 😅

**Pull request merged**